src/index.js

```diff
@@ -40,8 +40,15 @@ export default class DragDrop {
       settingsButton.setAttribute('draggable', 'true');
       settingsButton.addEventListener('dragstart', () => {
         this.startBlock = this.api.getCurrentBlockIndex();
+
+        // Here we clone the original item when we start the drag
+        if (this.holder.children[0].children[0].children[this.startBlock]) {
```

Comment:
@santiagoboterokommit This code can be improved
* check if there is an editor.js method that extracts the children you are looking for.
* if not, you can set a const, even a high-order const to be reused. 
* use object destruct to simplify.
* what will happen if for some reason any of the children do not exist?